src/enkf/gridio_gfs.f90

```diff
@@ -185,6 +185,7 @@ subroutine readgriddata_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,ntimes, &
   sst_ind = getindex(vars2d, 'sst')
   use_full_hydro = ( ql_ind > 0 .and. qi_ind > 0  .and. &
                      qr_ind > 0 .and. qs_ind > 0 .and. qg_ind > 0 )
+  use_full_hydro = .false.
 
```

Comment:
@emilyhcliu As this is basically a placeholder to allow for future development can you put a comment in here to explain why this is hard-coded?

Maybe comment out lines 186-7 as they are superfluous?